### PR TITLE
Feature/show2 969 nft media on web

### PR DESCRIPTION
**Pull request by alantoa**
# Why
- we hope media need to fill up the container on the mobile web, just like Native.

## Before 

https://user-images.githubusercontent.com/37520667/177353819-58dfebf1-75c1-454f-b76c-f68b5f6d9e56.mov

## After 

https://user-images.githubusercontent.com/37520667/177356499-d3e7cfab-d7ec-48f0-b9ee-743cd111183b.mov

# How

- optimized media's layout.

# Test Plan

check if the home NFT layout is looking good on the mobile web.

### CODE REVIEW

File: packages/app/components/swipe-list.tsx [hunk 1]
```diff
@@ -279,6 +279,7 @@ export const FeedItem = memo(
     itemHeight: number;
     listId?: number;
   }) => {
+    const [detailHeight, setDetialHeight] = useState(0);
```
Comment (axeldelafosse) [inline packages/app/components/swipe-list.tsx]: `setDetialHeight` -> `setDetailHeight`

File: packages/app/components/swipe-list.tsx [hunk 2]
```diff
@@ -429,6 +430,14 @@ export const FeedItem = memo(
               detailStyle,
               { bottom: bottomMargin },
             ]}
+            onLayout={({
+              nativeEvent: {
+                layout: { height },
+              },
+            }) => {
+              if (Platform.OS !== "web") return;
+              setDetialHeight(height);
```
Comment (axeldelafosse) [inline packages/app/components/swipe-list.tsx]: `setDetialHeight` -> `setDetailHeight`

**Review by axeldelafosse — APPROVED** [review]
LFG ✨

File: packages/app/components/swipe-list.tsx (hunk 1 above)
Comment (alantoa) [reply packages/app/components/swipe-list.tsx]: Sorry,I haven't connected to the monitor recently, my eyes are not very good.😂

**Pull request merged**